app/controllers/notifications_controller.rb

```diff
@@ -186,6 +186,16 @@ def sync
     end
   end
 
+  def subscribe_repo
+    current_user.github_client.update_subscription("#{params[:repo]}", {subscribed: true})
+    head :ok
+  end
+
+  def unsubscribe_repo
+    current_user.github_client.delete_subscription("#{params[:repo]}")
```

Comment:
Same here, no need to interpolate the string here, you can replace `"#{params[:repo]}"` with `params[:repo]`

Reply:
Fixed